frontend/src/components/browse-sightings-page/MapPage.tsx

```diff
@@ -0,0 +1,43 @@
+import { Sighting } from "../../clients/apiClient";
+// import { SlippyMap, Marker, Label, InfoBox } from "react-slippy-map";
+import GoogleMapReact from "google-map-react";
+import { Marker } from "./marker";
+import React from "react";
+import { map } from "leaflet";
+import "./mapPage.scss";
+
+interface MapPageProps {
+  sightingList: Sighting[];
+}
+
+export const MapPage: React.FC<MapPageProps> = ({ sightingList }) => {
+  const points = sightingList.map((sighting, index) => {
```

Comment:
You're not using the `index` parameter:
```suggestion
  const points = sightingList.map((sighting) => {
```